.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npx lint-staged -c .lintstagedrc.json
```

Comment:
```suggestion
npx lint-staged
```
I don't believe we need to specify the config here -- you're using a default filename for it, so it will automatically discover it